### PR TITLE
New options for Prometheus rw output

### PR DESCRIPTION
**Pull request by codebien**
Added the docs for the new available options of the Prometheus remote write output.

### CODE REVIEW

File: src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md [hunk 1]
```diff
@@ -158,14 +158,18 @@ k6 has special options for remote write output.
 | Name | Type |  Description |
 | ---- | ---- |  ----------- |
 | `K6_PROMETHEUS_RW_SERVER_URL` | `string`  | URL of the Prometheus remote write implementation's endpoint. Default is ``http://localhost:9090/api/v1/write`` |
-| `K6_PROMETHEUS_RW_HEADERS_<here-the-header-key>` | list of `string` | Additional headers to include in the HTTP requests. `K6_PROMETHEUS_RW_HEADERS_X-MY-HEADER=foo`|
-| `K6_PROMETHEUS_RW_USERNAME` | `string` | User for the HTTP Basic authentication at the Prometheus remote write endpoint. |
-| `K6_PROMETHEUS_RW_PASSWORD` | `string` | Password for the HTTP Basic authentication at the Prometheus remote write endpoint. |
-| `K6_PROMETHEUS_RW_PUSH_INTERVAL` | `string` | Interval of the metrics' aggregation and upload to the endpoint. Default is `5s` |
+| `K6_PROMETHEUS_RW_HEADERS_<here-the-header-key>` | list of `string` | Additional headers to include in the HTTP requests. `K6_PROMETHEUS_RW_HEADERS_X-MY-HEADER=foo`. |
```
Comment (olegbespalov) [inline src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md:161]:
```suggestion
| `K6_PROMETHEUS_RW_HEADERS_<here-the-header-key>` | list of `string` | An additional header to include in the HTTP requests. `K6_PROMETHEUS_RW_HEADERS_X-MY-HEADER=foo`. |
```

File: src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md [hunk 2]
```diff
@@ -158,14 +158,18 @@ k6 has special options for remote write output.
...
-| `K6_PROMETHEUS_RW_TREND_STATS` | list of `string` |  If Native Histogram is not enabled then it defines the stats functions to map for the all defined trend metrics. It's a comma-separated list of stats functions to include (e.g. `p(90),avg,sum`). Check the trend section to know the entire set of the supported stats. Default is `p(99)` |
+| `K6_PROMETHEUS_RW_TREND_STATS` | list of `string` |  If Native Histogram is not enabled then it defines the stats functions to map for the all defined trend metrics. It's a comma-separated list of stats functions to include (e.g. `p(90),avg,sum`). Check the trend section to know the entire set of the supported stats. Default is `p(99)`. |
 | `K6_PROMETHEUS_RW_INSECURE_SKIP_TLS_VERIFY` | `boolean` | If true, the HTTP client skips TLS verification on the endpoint. Default is `false`. |
 | `K6_PROMETHEUS_RW_STALE_MARKERS` | `boolean` | If true, the output at the end of the test marks all the seen time series as stale. Default is `false`. |
+| `K6_PROMETHEUS_RW_USERNAME` | `string` | User for the HTTP Basic authentication at the Prometheus remote write endpoint. |
```
Comment (olegbespalov) [inline src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md:168]:
```suggestion
| `K6_PROMETHEUS_RW_USERNAME` | `string` | Username for the HTTP Basic authentication at the Prometheus remote write endpoint. |
```

File: src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md [hunk 3]
```diff
@@ -158,14 +158,18 @@ k6 has special options for remote write output.
...
+| `K6_PROMETHEUS_RW_PASSWORD` | `string` | Password for the HTTP Basic authentication at the Prometheus remote write endpoint. |
+| `K6_PROMETHEUS_RW_CLIENT_CERTIFICATE` | `string` | Certifivate to authenticate via mTLS. |
+| `K6_PROMETHEUS_RW_CLIENT_CERTIFICATE_KEY` | `string` | Key to authenticate via mTLS. |
+| `K6_PROMETHEUS_RW_BEARER_TOKEN` | `string` | Set the Authorization header following the Bearer token format. |
```
Comment (olegbespalov) [inline src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md:172]: Maybe?
```suggestion
| `K6_PROMETHEUS_RW_BEARER_TOKEN` | `string` | A bearer token for authorization. |
```

File: src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md (hunk 1 above)
Comment (oleiade) [inline src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md:161]: I spotted a couple of things here:
1. we refer to the parameter as a list of `string`, but we mention, and give an example, of using only a single one. Maybe we could rephrase to outline what I understand as being is meant: multiple headers of the same format with different `<header-key>` can be set in the environment.
2. In the example the `<header-key>` is prefixed with `_X` but not in the description, was it on purpose? 

```suggestion
| `K6_PROMETHEUS_RW_HEADERS_X-<header-key>` | `string` | Additional headers to include in the HTTP requests can be set using this described format. e.g. `K6_PROMETHEUS_RW_HEADERS_X-MY-HEADER=foo`. |
```

What do you think?

File: src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md [hunk 4]
```diff
@@ -158,14 +158,18 @@ k6 has special options for remote write output.
...
+| `K6_PROMETHEUS_RW_TREND_STATS` | list of `string` |  If Native Histogram is not enabled then it defines the stats functions to map for the all defined trend metrics. It's a comma-separated list of stats functions to include (e.g. `p(90),avg,sum`). Check the trend section to know the entire set of the supported stats. Default is `p(99)`. |
 | `K6_PROMETHEUS_RW_INSECURE_SKIP_TLS_VERIFY` | `boolean` | If true, the HTTP client skips TLS verification on the endpoint. Default is `false`. |
 | `K6_PROMETHEUS_RW_STALE_MARKERS` | `boolean` | If true, the output at the end of the test marks all the seen time series as stale. Default is `false`. |
+| `K6_PROMETHEUS_RW_USERNAME` | `string` | User for the HTTP Basic authentication at the Prometheus remote write endpoint. |
+| `K6_PROMETHEUS_RW_PASSWORD` | `string` | Password for the HTTP Basic authentication at the Prometheus remote write endpoint. |
+| `K6_PROMETHEUS_RW_CLIENT_CERTIFICATE` | `string` | Certifivate to authenticate via mTLS. |
```
Comment (oleiade) [inline src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md:170]:
```suggestion
| `K6_PROMETHEUS_RW_CLIENT_CERTIFICATE` | `string` | Certificate to authenticate via mTLS. |
```

Corrects a typo in the word "certificate". On another note, I think "via" is a very latin-based language wording (we use extensively in french too), but I'm not sure it's that common in English though, @heitortsergent might know more 🤔

File: src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md (hunk 3 above)
Comment (oleiade) [reply src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md:172]: I would suggest

```suggestion
| `K6_PROMETHEUS_RW_BEARER_TOKEN` | `string` |  Sets the Authorization Bearer Token Header. |
```

File: src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md (hunk 2 above)
Comment (oleiade) [reply src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md:168]: Interestingly enough the [RFC](https://datatracker.ietf.org/doc/html/rfc7617) refers to the user name as "user-id". But I acknowledge that it's probably used as "username" all over the place regardless 😄

File: src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md [hunk 5]
```diff
@@ -158,14 +158,18 @@ k6 has special options for remote write output.
...
+| `K6_PROMETHEUS_RW_TREND_STATS` | list of `string` |  If Native Histogram is not enabled then it defines the stats functions to map for the all defined trend metrics. It's a comma-separated list of stats functions to include (e.g. `p(90),avg,sum`). Check the trend section to know the entire set of the supported stats. Default is `p(99)`. |
 | `K6_PROMETHEUS_RW_INSECURE_SKIP_TLS_VERIFY` | `boolean` | If true, the HTTP client skips TLS verification on the endpoint. Default is `false`. |
 | `K6_PROMETHEUS_RW_STALE_MARKERS` | `boolean` | If true, the output at the end of the test marks all the seen time series as stale. Default is `false`. |
+| `K6_PROMETHEUS_RW_USERNAME` | `string` | Username for the HTTP Basic authentication at the Prometheus remote write endpoint. |
+| `K6_PROMETHEUS_RW_PASSWORD` | `string` | Password for the HTTP Basic authentication at the Prometheus remote write endpoint. |
+| `K6_PROMETHEUS_RW_CLIENT_CERTIFICATE` | `string` | Certificate to authenticate via mTLS. |
+| `K6_PROMETHEUS_RW_CLIENT_CERTIFICATE_KEY` | `string` | Key to authenticate via mTLS. |
```
Comment (olegbespalov) [inline src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md:171]: What do you think about being more explicit that this is a path, not the data?
```suggestion
| `K6_PROMETHEUS_RW_CLIENT_CERTIFICATE` | `string` | A path to the PEM (Privacy-Enhanced Mail) formatted client certificate. |
| `K6_PROMETHEUS_RW_CLIENT_CERTIFICATE_KEY` | `string` | A path to the PEM formatted client private key. |
```

File: src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md (hunk 1 above)
Comment (codebien) [reply src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md:161]:
```suggestion
| `K6_PROMETHEUS_RW_HEADERS_<header-key>` | `string` | Additional headers to include in the HTTP requests can be set using this described format. For example, `K6_PROMETHEUS_RW_HEADERS_MY-HEADER-CUSTOM-KEY=foo`. |
```

File: src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md [hunk 6]
```diff
@@ -158,14 +158,18 @@ k6 has special options for remote write output.
...
+| `K6_PROMETHEUS_RW_BEARER_TOKEN` | `string` |  Sets the Authorization Bearer Token Header. |
```
Comment (codebien) [inline src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md:172]:
```suggestion
| `K6_PROMETHEUS_RW_BEARER_TOKEN` | `string` | Sets the Authorization Bearer Token Header. |
```

File: src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md [hunk 7]
```diff
@@ -158,14 +158,18 @@ k6 has special options for remote write output.
 | Name | Type |  Description |
 | ---- | ---- |  ----------- |
 | `K6_PROMETHEUS_RW_SERVER_URL` | `string`  | URL of the Prometheus remote write implementation's endpoint. Default is ``http://localhost:9090/api/v1/write`` |
-| `K6_PROMETHEUS_RW_HEADERS_<here-the-header-key>` | list of `string` | Additional headers to include in the HTTP requests. `K6_PROMETHEUS_RW_HEADERS_X-MY-HEADER=foo`|
-| `K6_PROMETHEUS_RW_USERNAME` | `string` | User for the HTTP Basic authentication at the Prometheus remote write endpoint. |
-| `K6_PROMETHEUS_RW_PASSWORD` | `string` | Password for the HTTP Basic authentication at the Prometheus remote write endpoint. |
-| `K6_PROMETHEUS_RW_PUSH_INTERVAL` | `string` | Interval of the metrics' aggregation and upload to the endpoint. Default is `5s` |
+| `K6_PROMETHEUS_RW_HEADERS_<header-key>` | `string` | Additional headers to include in the HTTP requests can be set using this described format. For example, `K6_PROMETHEUS_RW_HEADERS_MY-HEADER-CUSTOM-KEY=foo`. |
```
Comment (codebien) [inline src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md:161]:
```suggestion
| `K6_PROMETHEUS_RW_HEADERS_<header-key>` | `string` | An additional header to include in the HTTP requests can be set using this described format. For example, `K6_PROMETHEUS_RW_HEADERS_MY-HEADER-CUSTOM-KEY=foo`. |
```

File: src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md (hunk 1 above)
Comment (codebien) [reply src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md]: Resolved by https://github.com/grafana/k6-docs/pull/1366#discussion_r1350486668

**Review by olegbespalov — APPROVED** [review]
:rocket:

**Pull request merged**